### DISCUSSION TITLE
Custom module

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#17020] [How to]
When I create a new module, a table is created in the base, but nothing in the directory.

version 6.1.0

<img width="612" alt="Untitled" src="https://user-images.githubusercontent.com/75428743/230739438-5cf270dc-c2c5-4fce-a54f-a670e291b964.png">

**Comment by rskrzypczak** [comment]
You may have entered an incorrect module name.
In newer versions, the verification of the module name has been improved.
However, I would start with updating the system to a newer version ;)

**Reply by Akaki-1991** [reply]
> You may have entered an incorrect module name. In newer versions, the verification of the module name has been improved. However, I would start with updating the system to a newer version ;)

Not incorrect name.I tried with several names and it didn't create. He was creating from the beginning.
